packages/salesforcedx-apex-debugger/package.json

```diff
@@ -56,7 +56,7 @@
     "vscode-uri": "1.0.1"
   },
   "scripts": {
-    "bundle:debugger": "esbuild ./src/adapter/apexDebug.ts  --bundle --outfile=dist/apexdebug.js --format=cjs --platform=node --minify",
+    "bundle:debugger": "esbuild ./src/adapter/apexDebug.ts  --bundle --outfile=dist/apexdebug.js --format=cjs --platform=node --external:vscode --minify",
```

Comment:
I added the `--external:vscode` tag here because my Commit Workflows were failing with the following error after I added calls to telemetry:
![Screenshot 2024-03-29 at 11 13 10 AM](https://github.com/forcedotcom/salesforcedx-vscode/assets/139700604/1ddf18b9-340a-4ba3-9097-d192a8d43210)

I did a search on this error message and found https://github.com/evanw/esbuild/issues/103, which says to make the `vscode` module an external module for esbuild.  I found that the other `package.json` files in the repo used this tag to make `vscode` an external module, so I added the tag to this file, and the Commit Workflow passes now.